app/services/checkout/stripe_redirect.rb

```diff
@@ -12,10 +14,11 @@ def initialize(params, order)
     def path
       return unless stripe_payment_method?
 
-      payment = OrderManagement::Subscriptions::StripeScaPaymentAuthorize.new(@order).call!
+      payment = OrderManagement::Order::StripeScaPaymentAuthorize.new(@order).
+        call!(full_checkout_path, false)
```

Comment:
this boolean argument makes it hard to understand what it does and it's a code smell IMO